AppCenter/AppCenterTests/MSLogManagerDefaultTests.m

```diff
@@ -61,6 +61,68 @@ - (void)testInitNewChannel {
   assertThatUnsignedLong(channel.configuration.pendingBatchesLimit, equalToUnsignedLong(pendingBatchesLimit));
 }
 
+- (void)testProcessingLogWillFilterIfShouldFilter {
+
+  // If
```

Comment:
sounds like Given here, If is for conditional, I don't see any condition

Reply:
The condition is that the delegate returns `YES` or `NO`